go/workflow_test.go

```diff
@@ -0,0 +1,27 @@
+package app
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"go.temporal.io/sdk/testsuite"
+)
+
+func Test_Workflow(t *testing.T) {
+	// Set up the test suite and testing execution environment
+	testSuite := &testsuite.WorkflowTestSuite{}
+	env := testSuite.NewTestWorkflowEnvironment()
+
+	// Mock activity implementation
+	env.OnActivity(BookHotel, mock.Anything).Return(nil)
+	env.OnActivity(CancelHotel, mock.Anything).Return(nil)
```

Comment:
For this test, if nothing is failing, why do we need to put a expectation on the compensation activities?

Reply:
because I wasn't paying attention :-P fixed!